src/faucetConfig.ts

```diff
@@ -4,7 +4,13 @@ import botConfigSpec from "../env.bot.config.json";
 import serverConfigSpec from "../env.server.config.json";
 import { logger } from "./logger";
 
-type SpecType<T> = T extends { type: "string" } ? string : T extends { type: "number" } ? number : never;
+type SpecType<T> = T extends { type: "string" }
+  ? string
+  : T extends { type: "number" }
+  ? number
```

Comment:
that should go with additional tab indentation?

Reply:
It's auto-linted like that by our linting rules.